javascript/create_retirement_graph_service.js

```diff
@@ -186,4 +215,23 @@ FinancialFreedom.service('CreateRetirementGraphService', function() {
         parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",");
         return parts.join(".");
     };
-});
\ No newline at end of file
+
+    function findLabelCoordinates(graph_points) {
+
+        var labelX;
+        var expenses_label_y;
+        var income_label_y;
+
+        var findX = Math.round(graph_points.length * 0.25);
```

Comment:
call this something other than `findX`, it makes it seem like it's running a function or something. It appears to be more of an `x_index` or something along those lines.